.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: Build image
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build_and_push:
+    permissions:
+      packages: write
+      contents: read
+    uses: ericonr/test-repository/.github/workflows/eid.yml@559a8c8546ce5913e4c04606f3505766e2dd60ea
```

Comment:
Refactor to the epics-in-docker URI as soon as https://github.com/cnpem/epics-in-docker/pull/51 is merged.